### DISCUSSION TITLE
Audio in rtsp on HwV380E22_WF3_MINI_Q7_20201023

### DISCUSSION THREAD

**Discussion by MrTheFirst** [Arkady23/V380-Web#60] [Q&A]
Camera info:
Model: V380E3_C5#YW ([Ali](https://aliexpress.ru/item/4000429796084.html?sku_id=10000013757237002))
OS version: HwV380E22_WF3_MINI_Q7_20201023

In the application, broadcasting works with audio. There is no audio in the RTSP stream. I asked a respected Chinese seller a question, how can I get sound. To which I received a negative response:
`I'm very sorry, I consulted the technology and the camera does not have audio streaming`

There is sound in the application, so you can get it. Is it possible to improve your solution for transmitting audio from the camera?

P.S. ONVIF url from camera settings (http://192.168.*.*:8899/onvif/device_service) does not work. It doesn't seem to be supported on this model.

**Comment by Arkady23** [comment]
I know how to make sound. It is necessary to change the camera software. Remove the regular RTSP server and implement another one where there is sound. But I don't do that. I only change the settings that are provided by the existing manufacturer's software.
